components/python/rapidfuzz/rapidfuzz-PYVER.p5m

```diff
@@ -74,12 +74,12 @@ file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/_initialize_
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/_initialize_py.py
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/metrics_cpp.pyi
 file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/metrics_cpp.so
-file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/metrics_cpp_avx2.so
+file path=usr/lib/python$(PYVER)/vendor-packages/rapidfuzz/distance/metrics_cpp_avx2.so variant.arch=i386
```

Comment:
Please do not edit automatically generated files.  Such changes will be lost.

Reply:
How would one differentiate between architectures then, the sample-manifest is automatically created by: gmake sample-manifest and the purpose of the real manifest is there to exactly do such things - to differentiate, we used to do this for quite much non-python packages.

Reply:
Yes, it works like that (unfortunately) for non-python (and non-perl) packages, but not for python packages integrated using `python-integrate-project`.  I'm sorry, but such change will be lost during next rapidfuzz rebuild - when either new rapidfuzz is released or we add support for new Python major version.

To do tweaks like this properly you should change `python-integrate-project.conf` accordingly.  In this case you should look at `%hook-manifest%`.